entries/entry03.md

```diff
@@ -1,7 +1,7 @@
 # Entry 3
```

Comment:
Sources: 1
EDP: 0
Knowledge: 0 (code snippets? evidence of learning?)
Skills: 0
Professionalism: 1